sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/TestSuiteBase.java

```diff
@@ -1125,4 +1127,88 @@ protected int expectedNumberOfPages(int totalExpectedResult, int maxPageSize) {
     public static CosmosClientBuilder copyCosmosClientBuilder(CosmosClientBuilder builder) {
         return CosmosBridgeInternal.cloneCosmosClientBuilder(builder);
     }
+
+
+    public enum City {
+        NEW_YORK,
+        LOS_ANGELES,
+        SEATTLE
+    }
+
+    public final class Pet extends JsonSerializable {
```

Comment:
POJO end user type is not expected to extend JsonSerializable. Why are we extending the POJO test classes from POJOSerializable?

Reply:
Yup these will be refactored